projects/packages/config/changelog/Included the changes to configure the 'publicize' package.

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I'm sure it will work, but we might want to simplify this filename

Reply:
Ah, it was auto-generated, sorry about this monstrosity of a name. I have corrected it.